src/vue/forms/AssetCreateForm.vue

```diff
@@ -323,7 +324,9 @@ export default {
     }),
 
     assetRequestOpts () {
-      const requestId = this.request.id || ASSET_CREATION_REQUEST_ID
+      const requestId = this.request instanceof AssetCreateRequestRecord
```

Comment:
a bit confusing check. the `request` prop cannot be instance of another that `AssetCreateRequestRecord ` class, why just don't check for existence?

Reply:
Fixed, checked just for existence